pkg/gohai/processes/processes_test.go

```diff
@@ -0,0 +1,101 @@
+// This file is licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2014-present Datadog, Inc.
+
+//go:build linux || darwin
+
+package processes
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGet(t *testing.T) {
+	processes, err := Get()
+	require.NoError(t, err)
+
+	require.NotEmpty(t, processes)
+	for _, process := range processes {
+		assert.NotEmpty(t, process.Name)
+
+		assert.NotEmpty(t, process.Usernames)
+		assert.IsIncreasing(t, process.Usernames)
```

Comment:
Do we guarantee that usernames are sorted in `/pkg/gohai/processes/gops`  ?

Reply:
Yes, cf https://github.com/DataDog/datadog-agent/blob/main/pkg/gohai/processes/gops/process_group.go#L54.
It wasn't tested before so I added it to the test.

Reply:
Should this test be in `pkg/gohai/processes/gops` then ?

Reply:
I guess it depends on whether that guarantee is for `gops` specifically or the full processes package, ie. "Should users of the package assume that the usernames are sorted ?"
As I specified in the doc of the `ProcessGroup` type (which is the exported type of the package) that the usernames are sorted, I think we can keep that at the package level